drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,31 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DC_STM
+	bool
+	prompt "USB device controller driver for STM32 devices"
+	depends on HAS_STM32CUBE
+	default n
+	help
+	STM32 family USB device controller Driver.
+
+config USB_DC_STM_EP_NUM
+	int
+	depends on USB_DC_STM
+	default 4
+
+config USB_DC_STM_FIFO_WORD_NUM
```

Comment:
Could we use the PACKET_RAM_SIZE instead of FIFO words? SOCs with USB device controller doesn't have FIFOs. It would make it easier to add support for these SOCs later.

Reply:
Not sure, because we really use this define to configure FIFOs via the HAL.

Reply:
We could calculate the FIFOs words from RAM SIZE (FIFO_words = RAM_size / 4) in the code.